src/Sylius/Behat/Page/Shop/Cart/SummaryPage.php

```diff
@@ -264,7 +264,7 @@ protected function getDefinedElements()
             'cart_items' => '#sylius-cart-items',
             'cart_total' => '#sylius-cart-button',
             'clear_button' => '#sylius-cart-clear',
-            'coupon_field' => '#sylius_cart_promotionCoupon',
+            'coupon_field' => '#sylius_order_promotionCoupon',
```

Comment:
Since all other selectors are using `cart` we should leave this one also, at least for now.